ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/SketchOnMap.qml

```diff
@@ -15,11 +15,14 @@
 // [Legal]
 
 import QtQuick 2.6
-import QtQuick.Controls 2.2
+import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.15
 import Esri.Samples 1.0
 
 Item {
+
+    property var drawStatus: ""
```

Comment:
drawStatus could be a QML-side enum instead of a string - https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#enumeration-attributes